apps/music/js/ui/views/search_view.js

```diff
@@ -127,7 +127,7 @@ var SearchView = {
 
     if (this.searchContext === this.context.ALL ||
         this.searchContext === this.context.SONGS) {
-      this.searchHandles.title = musicdb.enumerate(
+      this.searchHandles.title = Database.enumerate(
         'metadata.title',
```

Comment:
Looks like after we wrapped the MediaDB into the Database object, we need to fill the `range` and `direction` arguments or there will be an error says:

`[JavaScript Error: "DataError: Data provided to an operation does not meet requirements." {file: "app://music.gaiamobile.org/shared/js/mediadb.js" line: 1170}]`

when music is searching in all songs view.